packages/@ember/-internals/glimmer/tests/integration/application/rendering-test.js

```diff
@@ -70,7 +128,7 @@ moduleFor(
       });
     }
 
-    ['@test it can render a nested route']() {
+    ['@feature(!EMBER_ROUTING_MODEL_ARG) it can render a nested route']() {
```

Comment:
IMHO, this shouldn't be feature flagged. This test confirms `this.model` continues to work, and should stay around even after the feature flag is removed.

Reply:
I think we said we are fine testing the happy path (`@model` going forward) in these tests (there is a copy of the test below in the new style) and just add some targeted tests to avoid regression, which I did in the latest commit, if you still think this or any other tests specifically  need to be test both ways, feel free to comment again.

Reply:
Nope, this is great, thank you!